test/blocks/bulk-publish/bulk-publish-utils.test.js

```diff
@@ -126,7 +126,7 @@ describe('Bulk preview and publish', () => {
     localStorage.clear();
   });
 
-  it('Bulk preview URLs', async () => {
+  it.skip('Bulk preview URLs', async () => {
```

Comment:
Is this still needed, is there any plan to enhance it or can it be removed entirely?

Reply:
@chrischrischris You mentioned that this test is intermittently failing. Can you please point me to links where I can see the failures? I don't see any failures when I run it locally.

Reply:
I can't repro it now, so removing the `.skip`

Reply:
Let's have an eye on it and ping @jckautzmann if we see it again so he can take a look